plugin/Utils.js

```diff
@@ -16,6 +16,25 @@ class Utils {
     static isEmpty(val) {
         return !val || !val.length;
     }
+
+    /**
+     * Returns true if argument is NOT object
+     * @param val
+     * @returns {boolean}
+     */
+    static isNotObject(val) {
+        return !Utils.isObject(val);
+    }
+
+    /**
+     * Returns true if argument is object
+     * @param val
+     * @returns {boolean}
+     */
+    static isObject(val) {
+        const type = typeof val;
+        return val !== null && (type === 'object');
```

Comment:
From my perspective it would be better to create special function (for example: **isNotEmptyObject**) and remove from the **isObject** function the null checking. If JS specification indicates that typeof null === "object" I think that **isObject** function should work consistently

Reply:
I think this concern is applicable in case of *isObjectType* naming (because type of null is "object") but here data structure considered by itself. Also this function is taken from lodash module